src/server/api/routes/users/v2/follow.js

```diff
@@ -0,0 +1,91 @@
+const { Router } = require('express');
+const {
+  authenticateV2Login,
+  checkWalletMatch,
+} = require('../../../middleware/auth');
+const { setPrivateCacheHeader } = require('../../../middleware/cache');
+const { handleRestfulError } = require('../../../middleware/error');
+const { isValidAddress } = require('../../../util/cosmos');
+const {
+  FieldValue,
+  walletUserCollection,
+} = require('../../../../modules/firebase');
+
+const router = Router();
+
+router.get('/v2/users/:wallet/followers', async (req, res, next) => {
```

Comment:
Is this info needed to be public in design?

Reply:
Not sure about this, if not, we can just return followers & followers in `/users/min`?

Reply:
please go verify if there is any ui element that requires listing of this info in non-setting related page